### PR TITLE
Mocktic's card

### PR DESCRIPTION
**Pull request by Mocktic**
added my name's card code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,39 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+
```
Comment (lax0248917) [inline index.html:118]: Only need one line between the instructional line and the comment line that starts your card. Please remove the additional line.

File: index.html [hunk 2]
```diff
@@ -115,6 +115,39 @@ <h2>A project for first-time contributions</h2>
...
+<!-- ________ Mocktic Joshi's Contributor card START ________  -->
```
Comment (lax0248917) [inline index.html:119]: Tab this comment start to be inline with your `<div>` tag

File: index.html [hunk 3]
```diff
@@ -115,6 +115,39 @@ <h2>A project for first-time contributions</h2>
...
+        <div class="card">
+          <p class="name">Mocktic Joshi</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/mocktic" target="_blank">mocktic</a>
+          </p>
+          <p class="about">I love to code and build something fot the community</p>
```
Comment (lax0248917) [inline index.html:126]: Found a typo in this sentence. I think you meant to use the word "for" not "fot" :)

File: index.html [hunk 4]
```diff
@@ -115,6 +115,39 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="Third resource">
+                  W3schools
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+<!-- ________ Mocktic Joshi's Contributor card ENDS ________  -->
```
Comment (lax0248917) [inline index.html:148]: Place this comment line in line with your closing `<div>` tag, and make sure that there is only one line space between this comment line and the comment line of the next contributor's card. Suggest running `npm run prettier-html` on your code to format it correctly. Let us know if you have any questions.

**Review by lax0248917 — CHANGES_REQUESTED** [review]
Hello @Mocktic, thank you for contributing. I left you a couple comments on some lines of code that need to be changed. Let me know if you have any questions. Make those changes and push your commit to this PR and we will get your card added. Take care 🚀

**Comment by Mocktic** [comment]
Hi @lax0248917, Thanks for reviewing my code and suggesting ways to make the code better. I've made all the changes, still, if I left something out please let me know I'll definitely going to fix it. This was my second contribution to a project on Github and I really enjoyed contributing. Thanks :)

**Review by lax0248917 — APPROVED** [review]
This all looks great. Thank you for making those changes, I will get this merged right away. 🎉

**Pull request merged**